gitlab-runner/src/lib.rs

```diff
@@ -31,10 +32,23 @@
 //!
 //! #[tokio::main]
 //! async fn main() {
-//!     let mut runner = Runner::new(
-//!         "https://gitlab.example.com".try_into().unwrap(),
-//!         "runner token".to_owned(),
-//!         PathBuf::from("/tmp"));
+//!     let (layer, jobs) = GitlabLayer::new();
+//!     tracing_subscriber::Registry::default()
+//!        .with(
+//!           tracing_subscriber::fmt::Layer::new()
```

Comment:
Should be able to just be:

```suggestion
//!           tracing_subscriber::fmt::layer()
```